package.json

```diff
@@ -91,6 +100,7 @@
     "marko-widgets": "^6",
     "mobile-detect": "^1.4.1",
     "mocha": "^5.1.1",
+    "moment": "^2.22.2",
```

Comment:
We get some dependency error if we don't include this. Even [upgrading to v4.0.0-alpha.10 didn't help](https://github.com/storybooks/storybook/issues/3901) (may this will be fixed in v4.0.0). But for now, this seems to be required.